### PR TITLE
Pull request for gslab-python/#41: Stata flavor

### PR DESCRIPTION
**Pull request by stanfordquan**
We had complicated logic to look for the user's Stata executable that was triggered if the user enters "None", but we set the default argument to "StataMP", thus bypassing the whole complicated logic bit. 
Now the default option is "None", we get to use the complicated logic.

Tested on both Windows and Unix, with or without default option.

In particular, on Windows, the user can specify the path for %STATAEXE% and call `scons` without `sf` option. Scons will know to run `%STATAEXE% blah blah`.

### CODE REVIEW

**Comment by stanfordquan** [comment]
@arosenbe can you please kindly review? Thanks!

File: output/analysis/plot.eps [hunk 1]
```diff
@@ -385,18 +385,47 @@
 } def
 /Helvetica /Helvetica-0 MF-Helvetica
 /Slw 0.120 def
-1.000 1.000 1.000 setrgbcolor
+0.882 0.902 0.941 setrgbcolor
```
Comment (arosenbe) [inline output/analysis/plot.eps]: I think you need to change the default back to s1mono. The color in this eps is not standard.

File: sconstruct.log [hunk 2]
```diff
@@ -1,4 +1,8 @@
 scons: done reading SConscript files.
 scons: Building targets ...
-scons: `.' is up to date.
+build_r(["output/data/data.txt"], ["source/data/build.R"])
```
Comment (arosenbe) [inline sconstruct.log]: How about you rerun before committing to show that everything is up-to-date.

File: output/analysis/plot.eps (hunk 1 above)
Comment (stanfordquan) [inline output/analysis/plot.eps]: done

File: sconstruct.log (hunk 2 above)
Comment (stanfordquan) [inline sconstruct.log]: good catch! thankfully everything works

**Pull request merged**